src/SonarQube.Common/AnalysisConfig/AnalysisConfig.cs

```diff
@@ -62,6 +62,8 @@ public class AnalysisConfig
 
         public string SonarQubeHostUrl { get; set; }
 
+        public string SonarQubeVersion { get; set; }
```

Comment:
SonarQube Version is now a first-class property of the `AnalysisConfig` for the sake of simplicity.